libraries/AP_Proximity/AP_Proximity_Backend.h

```diff
@@ -20,6 +20,9 @@
 #include <AP_Common/Location.h>
 #include "AP_Proximity_Boundary_3D.h"
 
+#define PROXIMITY_GND_DETECT_THRESHOLD 1.0f // set ground detection threshold to be 1 meters
+#define PROXIMITY_ALT_DETECT_TIMEOUT_MS 500 // alt readings should arrive within this much time
```

Comment:
Not too important but I guess this timeout is for the rangefinder?  Maybe we should call it PROXIMITY_RNGFND_TIMEOUT_MS?

Reply:
I left this one as it is because I don't want some new developer to be confused with the forward facing rangefinder driver that is also part of the proximity lib. 
Although the only source of alt readings for the proximity lib is downward facing rangefinder, but who knows, if sometime we change it.. probably with the 3D cameras then this name won't be valid. 

But i'll be happy to rename if you still think its a good idea. :)